analyzers/tests/SonarAnalyzer.UnitTest/Rules/Utilities/CopyPasteTokenAnalyzerTest.cs

```diff
@@ -72,22 +72,23 @@ public class CopyPasteTokenAnalyzerTest
         [TestMethod]
         public void Verify_Duplicated_CS_GlobalUsings()
         {
-            var testRoot = Root + TestContext.TestName;
-            var fileName = "Duplicated.CSharp10.cs";
-            OldVerifier.VerifyNonConcurrentUtilityAnalyzer<CopyPasteTokenInfo>(
-                new[] { Root + fileName },
-                new TestCopyPasteTokenAnalyzer_CS(testRoot, false),
-                @$"{testRoot}\token-cpd.pb",
-                TestHelper.CreateSonarProjectConfig(testRoot, ProjectType.Product),
-                messages =>
-                {
-                    messages.Should().HaveCount(1);
-                    var info = messages.Single();
-                    info.FilePath.Should().Be(fileName);
-                    info.TokenInfo.Should().HaveCount(39);
-                    info.TokenInfo.Where(x => x.TokenValue == "$num").Should().HaveCount(2);
-                },
-                options: ParseOptionsHelper.FromCSharp10);
+            const string testRoot = BasePath + nameof(Verify_NotRunForTestProject_CS);
+            const string fileName = "Duplicated.CSharp10.cs";
+
+            verifierBuilder
+                .AddAnalyzer(() => new TestCopyPasteTokenAnalyzer_CS(testRoot, false))
+                .AddPaths(fileName)
+                .WithOptions(ParseOptionsHelper.FromCSharp10)
+                .WithSonarProjectConfigPath(TestHelper.CreateSonarProjectConfig(testRoot, ProjectType.Product))
+                .WithProtobufPath(@$"{testRoot}\token-cpd.pb")
+                .VerifyUtilityAnalyzer<CopyPasteTokenInfo>(messages =>
```

Comment:
These few lines (verifier creation) are repeated for a couple of times. WDYT about creating a UtilityVerifier, that would remove this duplication?

Reply:
I thought about this and I think that it will be one more level of indirection which I would like to avoid for now. We have now the builder which is responsible to build a verifier. 

I'm reticent in adding a `UtilityVerifier` which will call `VerifierBuilder` to create a `Verifier`.